### PR TITLE
Add `init` field to v2 schema

### PR DESCRIPTION
**Pull request by imikushin**

### CODE REVIEW

**Comment by imikushin** [comment]
Address rancher/rancher#8608

File: convert/convert.go [hunk 1]
```diff
@@ -258,6 +258,7 @@ func Convert(c *config.ServiceConfig, ctx project.Context) (*container.Config, *
 		DNS:         utils.CopySlice(c.DNS),
 		DNSOptions:  utils.CopySlice(c.DNSOpt),
 		DNSSearch:   utils.CopySlice(c.DNSSearch),
+		Init:        utils.BoolCopyPtr(c.Init),
```
Comment (joshwget) [inline convert/convert.go]: `&[]bool{c.Init}[0]` should work here.

Comment (imikushin) [reply convert/convert.go]: good one

Comment (imikushin) [reply convert/convert.go]: fixed

**Comment by joshwget** [comment]
Assuming tests will fail until Cattle supports it. We can leave this open until we can bump the Cattle version. LGTM otherwise.

**Comment by deniseschannon** [comment]
Closed in favor of https://github.com/rancher/rancher-compose-executor/pull/67

**Pull request closed without merging**